openapi/workflow_execution_service.swagger.yaml

```diff
@@ -100,22 +100,22 @@ paths:
         - WorkflowExecutionService
     post:
       summary: |-
-        Run a workflow, this endpoint will allow you to create a new workflow request and
+        Submit a workflow job, this endpoint will allow you to create a new job request and
```

Comment:
As there is likely a queued period before the job gets run, so `submit` seems better reflect the situation.